modify import.js again

Kyubinhan opened the pull request:
Add ClientAgreement model
modify updateClient method in order to correctly insert into client_agreement table

jleach commented on .babelrc:
```diff
@@ -4,7 +4,8 @@
       }
-    }]
+    }],
+    "stage-3"
```
@Kyubinhan What feature is this needed for?

jleach commented on package.json:
```diff
@@ -48,6 +48,7 @@
     "babel-preset-airbnb": "^2.4.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
```
If we're not including stage-2 in .babelrc can we remove this package?

jleach commented on server/router/routes/agreement.js:
```diff
@@ -147,7 +68,7 @@ const transformClients = (clients, clientTypes) => {
       const ctype = clientTypes.find(t => t.id === c.clientAgreement.clientTypeId);
       delete client.clientAgreement;
-      return Object.assign(client, { clientTypeCode: ctype.code });
+      return { ...client, clientTypeCode: ctype.code };
```
👍

jleach commented on server/router/routes/agreement.js:
```diff
@@ -189,9 +110,9 @@ const transformAgreement = (agreement, clientTypes) => {
 
 // Get all agreements
 router.get('/', asyncMiddleware(async (req, res) => {
-  const { term = '', limit = 10, page } = req.query;
```
I think the limit should be moved to constants.js.

jleach commented on server/router/routes/agreement.js:
```diff
@@ -220,17 +141,19 @@ router.get('/', asyncMiddleware(async (req, res) => {
       // Its a bit tough to get the count from Sequlize but a raw query works great. A where clause
       // is applied only if the user is not an administrator.
-      let query = 'SELECT count(*) FROM agreement JOIN ref_zone ON agreement.zone_id = ref_zone.id';
+      let query = `SELECT count(*) FROM agreement JOIN ref_zone ON agreement.zone_id = ref_zone.id WHERE forest_file_id LIKE '%${term}%'`;
```
This works when term is undefined?

jleach commented on server/router/routes/includes.js:
```diff
@@ -0,0 +1,130 @@
+// a class for managing all the include attributes for models in one place
```
- Please add the standard header to all new files.
- No `'use strict'`?

Commit:
modify import.js again

Add ClientAgreement model
modify updateClient method in order to correctly insert into client_agreement table

jleach commented on scripts/import.js:
```diff
@@ -114,16 +117,19 @@ const updateZone = async (data) => {
           description: zoneDescription || 'No description available',
           district_id: district.id,
-          user_id: staff && staff.id,
+          userId: staff && staff.id,
```
Why is this one camel case while `district_id` is snake notation?

Commit:
address pr comment
